static/app/data/platformCategories.tsx

```diff
@@ -293,11 +293,40 @@ export const releaseHealth: PlatformKey[] = [
   'native-qt',
 ];
 
-export const replayPlatforms: readonly PlatformKey[] = [
+// These are the backend platforms that can set up replay -- e.g. they can be set up via a linked JS framework or via JS loader.
+// Note: not currently comprehensive -- only included ones that are also in the `backend` platforms listed above
+// TODO: add all the platforms
+const replayBackendPlatforms: readonly PlatformKey[] = [
+  'bun',
+  'dotnet-aspnetcore',
+  'dotnet-aspnet',
+  'elixir',
+  'java-spring',
+  'java-spring-boot',
+  'node',
+  'node-express',
+  'php',
+  'php-laravel',
+  'php-symfony',
+  'python-aiohttp',
+  'python-bottle',
+  'python-django',
+  'python-falcon',
+  'python-fastapi',
+  'python-flask',
+  'python-pyramid',
+  'python-quart',
+  'python-sanic',
+  'python-starlette',
+  'python-tornado',
+  'ruby-rails',
+];
+
+// These are the frontend platforms that can set up replay.
+const replayFrontendPlatforms: readonly PlatformKey[] = [
   'capacitor',
   'electron',
   'javascript-angular',
-  // 'javascript-angularjs', // Unsupported, angularjs requires the v6.x core SDK
```

Comment:
you wanna remove this? i put it in (a long time ago) as a note, in case people are like "why is angularjs missing, whats the difference"